src/assets/githubAction/githubActionBeginner.md

```diff
@@ -0,0 +1,130 @@
+---
+title: GitHub Actions - CI & CD
+authorName: Nabin Ale
+authorAvatar : https://avatars.githubusercontent.com/u/61624650?s=400&u=8f992a9ca189af19c40cb070a5af1e02669d3884&v=4
+authorLink: https://github.com/nabim777
+createdAt: Oct 30, 2023
+tags: CI, CD, github, githubAction
```

Comment:
```suggestion
tags: CI, CD, github, Github Actions
```

Reply:
I think spaces are not included in tags

Reply:
@SagarGi   I think we don't use space in between for tag according to the other site tag I have seen example:fb, youtube, twitter, instagram?